pkg/config/loader.go

```diff
@@ -49,15 +50,21 @@ type Loader struct {
 	// The Kubernetes client that will be used to load Kubernetes objects:
 	client kubernetes.Interface
 
-	// The names of the configuration files, in the order that they should be loaded:
+	// The names of the configuration directories and files, in the order that they should be loaded:
 	files []string
 
+	// The names of loaded configuration files:
+	LoadedFiles []string
```

Comment:
Is it necessary to make `LoadedFiles` public?

Reply:
thanks ! it was when the logic was in healer ... now it's not :+1: